frontend/src/metabase/visualizations/components/ChartTooltip/EChartsTooltip/EChartsTooltip.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import { SdkVisualizationWrapper } from "__support__/storybook";
+import { type MetabaseTheme, defineEmbeddingSdkTheme } from "embedding-sdk";
+import { Flex, Tooltip } from "metabase/ui";
+
+import { EChartsTooltip, type EChartsTooltipProps } from "./EChartsTooltip";
+
+const data: EChartsTooltipProps = {
+  header: "Header Text",
+  rows: [
+    { name: "Foo", values: ["Foo Row"] },
+    { name: "Focused", values: ["Focused Row"], isFocused: true },
+  ],
+};
+
+const DefaultTemplate = () => (
+  <Flex align="center" justify="center" p="xl" h="400px">
+    <Tooltip label={<EChartsTooltip {...data} />} opened>
+      <div>Tooltip Target</div>
+    </Tooltip>
```

Comment:
We use the Mantine Tooltip as a container to simulate how it would look like in EChart's Tooltip, so we can add visual tests in Storybook. I think the tooltip is probably from echarts core, so I'm not sure how to actually bring it here.